requirements.txt

```diff
@@ -1,11 +1,11 @@
 glfw==2.5.0
 GPy==1.10.0
-GPyOpt @ git+git://github.com/yunshengtian/GPyOpt@5fc1188ffdefea9a3bc7964a9414d4922603e904
+GPyOpt @ git+https://github.com/yunshengtian/GPyOpt@5fc1188ffdefea9a3bc7964a9414d4922603e904
```

Comment:
Can we make the same changes to `environment.yml`? Same as below.

Reply:
Yes, I will fix it.

Reply:
https://github.com/EvolutionGym/evogym/pull/25/commits/76a041a598af1a91adb1e054b96b20ea33ecac43